src/components/PublishedArticles/PublishedArticles.js

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment } from 'react';
+import styles from '../UserProfileTabs/UserProfileTabs.module.scss';
+import apple from '../../assets/images/apple.jpg';
+import andela from '../../assets/images/andela.png';
```

Comment:
I think we should be more specific with imports naming. why not `andelaIcon` or `andelaLogo`instead of just `andela`

Reply:
Okay. I will work on this too